addon/components/draggable-object.js

```diff
@@ -31,26 +25,27 @@ export default Ember.Component.extend({
   },
 
   didInsertElement() {
-    Ember.run.scheduleOnce('afterRender', ()=> {
-      let self = this;
```

Comment:
This addon seems to use arrow functions throughout. Using them here should remove the need for assigning `self`